### DISCUSSION TITLE
Currency Paypal Donations

### DISCUSSION THREAD

**Discussion by souzadavi** [impress-org/givewp#5857] [Developer Q&A] [answered]
There is a global variable that make currency globally, We need change the currency, but the Paypal SDK is always getting this variable and not the variable is defined over form.

![121951946-f7d5af00-cd31-11eb-8c8b-3551ddb26cf7](https://user-images.githubusercontent.com/636225/123158591-60afdc00-d442-11eb-9a1b-113648e00d30.png)
![121951925-f1473780-cd31-11eb-98b1-06ae9a074179](https://user-images.githubusercontent.com/636225/123158595-61487280-d442-11eb-8b30-f3fc3abc87c7.png)
![121951084-d9bb7f00-cd30-11eb-84de-e7e4e42a3f85](https://user-images.githubusercontent.com/636225/123158596-61e10900-d442-11eb-9649-d62696a8162f.png)

Is it clear the trouble?

Thanks

**Comment by ravinderk** [comment, marked as answer]
@souzadavi You can use [`give_form_html_tags`](https://github.com/impress-org/givewp/blob/5ff41246104e1bfe1cdc414400f8f3841a96fa71/includes/forms/template.php#L2338) filter to overwrite currency in the form's data attribute.

@souzadavi I curious to know why you are seeing two different currencies in the donation form and javascript global param. Can tell us about the global currency code. is it `USD` or `BRL`? Are you writing custom code to support multiple currencies?
